### PR TITLE
Remove BeforeTestWithContext, AfterTestWithContext

### PR DESCRIPTION
**Pull request by findepi**
They were left for backwards compatibility reasons. However, due to a coding mistake, they didn't work (details: https://github.com/trinodb/tempto/pull/113/files#r1211056558).

Remove the broken annotations to prevent users from trying to use them.

### CODE REVIEW

File: tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java [hunk 1]
```diff
@@ -17,9 +17,7 @@
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 import io.trino.tempto.AfterMethodWithContext;
-import io.trino.tempto.AfterTestWithContext;
```
Comment (Praveen2112) [inline tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java:20]: Shouldn't the maven compiler throws unused import error ?

**Review by Praveen2112 — APPROVED** [review]

File: tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java (hunk 1 above)
Comment (findepi) [reply tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java:20]: in trino checkstyle is responsible for that

Comment (Praveen2112) [reply tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java:20]: Any reason for not using checkstyle plugin here ?

Comment (findepi) [reply tempto-examples/src/main/java/io/trino/tempto/examples/SimpleQueryTest.java:20]: no reason that i am aware of

**Pull request merged**